server/src/com/thoughtworks/go/config/CachedFileGoConfig.java

```diff
@@ -95,9 +96,12 @@ public void onTimer() {
 
     private synchronized void loadFromDisk() {
```

Comment:
:)

Well, this needs more thought before removing so many synchronized keywords. Let's delay that. What you've done looks correct, but I don't want to rush it.